erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -810,7 +810,8 @@ def update_item(source_doc, target_doc, source_parent):
 				"parent": "against_sales_invoice",
 				"serial_no": "serial_no",
 				"sales_order": "against_sales_order",
-				"so_detail": "so_detail"
+				"so_detail": "so_detail",
+				"cost_center": "cost_center"
```

Comment:
It should not set cost center in DN based on sales invoice. In sales invoice, default cost center is set based on default selling cost center in item and it is used for booking income. But in Delivery note, it should use default buying cost center from Item, as it is used to book COGS (expense).